docker-compose-dev.yml

```diff
@@ -1,16 +1,47 @@
-version: "3.9"
+version: "2.4"
 
 services:
 
+  dev_dataverse:
+    container_name: "dev_dataverse"
+    hostname: dataverse
+    image: gdcc/dataverse:unstable
+    restart: on-failure
+    user: payara
+    environment:
+      - DATAVERSE_DB_HOST=postgres
+      - DATAVERSE_DB_PASSWORD=secret
+    ports:
+      - "8080:8080" # HTTP
+      - "9009:9009" # JDWP
+      - "8686:8686" # JMX
+    networks:
+      - dataverse
+    depends_on:
+      - dev_postgres
+      - dev_solr
+    volumes:
+      - ./docker-dev-volumes/app/data:/dv
+      - ./docker-dev-volumes/app/secrets:/secrets
+    tmpfs:
+      - /dumps:mode=770,size=2052M,uid=1000,gid=1000
+      - /tmp:mode=770,size=2052M,uid=1000,gid=1000
+    mem_limit: 2147483648 # 2 GiB
+    mem_reservation: 1024m
+    privileged: false
+
   dev_postgres:
```

Comment:
Could we reuse the health checks Patrick Carlson added within his MR? 

https://github.com/carlsonp/dataverse/blob/134695d3ba2262ac93f66b02bc5a33875a63df31/docker-compose.yml

Reply:
I like the idea! This is somewhat extending the scope of this PR/issue, but seems reasonable for me. Thanks for the hint.

Reply:
I vote next PR. Merge it, merge it! 🚀 :shipit: